os/arch/arm/src/amebad/amebad_enet.c

```diff
@@ -211,10 +215,16 @@ void up_netinitialize(void)
 		rtw_printf("Failed to register amebad netdev\n");
 	}
 #endif
+#ifndef CONFIG_VIRTUAL_BLE
```

Comment:
How about changing like below?
```
#ifdef CONFIG_VIRTUAL_BLE
    vble_start();
#else
#if defined(CONFIG_AMEBAD_BLE) && defined(CONFIG_DRIVERS_BLE)
	ameba_bm_dev_ble0 = bledev_register(&g_trble_drv_ops);
	if (ameba_bm_dev_ble0 == NULL) {
		rtw_printf("Failed to register amebad netdev\n");
	}
#endif
#endif
```
Sometimes, `ifndef` can make confusion, because many people can mistake.
So it is better to change `ifdef~else~endif` for easy understanding.

Reply:
Done